source/dom/platform.js

```diff
@@ -45,6 +45,8 @@ enyo.platform = {
 		{platform: "androidChrome", regex: /Android .* Chrome\/(\d+)[.\d]+/},
 		// Android 2 - 4
 		{platform: "android", regex: /Android (\d+)/},
+		// Some android phones, e.g. ZTE and Lenovo
+		{platform: "android", regex: /Android\/(\d+)/},
```

Comment:
Is it better to have a separate entry for "android" or simply modify the previous entry with something like `/Android(?:\/)?(\d+)/ `? Wouldn't that do the same thing?